CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ CHANGELOG
 - Allow for shorthand property values instead of duplicating project, zone, etc.
   [#56](https://github.com/pulumi/pulumi-google-native/issues/56)
 
-- Populate `project` properties automatically from a config value
+- Populate `project`, `location`, `zone` properties automatically from config values
```

Comment:
And region?

Reply:
It's a bit odd because the `region` config goes into `location` properties. I haven't seen `region` properties yet.